test/runner.py

```diff
@@ -2,6 +2,9 @@
 import os
 import sys
 
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
This hack is not (really) related to the one in `setup.py`, I think...

This one is for the `import test`s below, the one in `setup.py` is for `import vsc.install` (since the latter insert `.../lib`)